docs/known-issues.md

```diff
@@ -22,6 +22,184 @@ await ctx.Database.EnsureDeletedAsync();
 await ctx.Database.EnsureCreatedAsync();
 ```
 
+### CockroachDB doesn't support concurrency token column
+Github issue: https://github.com/cockroachdb/cockroach/issues/33296
 
+Entity Framework Core supports the concept of optimistic concurrency - a property on your entity is designated as a concurrency token, and EF Core detects concurrent modifications by checking whether that token has changed since the entity was read.
+
+In PostgreSQL, the xmin column, which xmin holds the ID of the latest updating transaction, can be used as concurrency token column.
+
+### CockroachDB doesn’t support such a column.
```

Comment:
nit: this should not have `###` before it, since it seems like a subsection of the previous one.